client/my-sites/theme/main.jsx

```diff
@@ -227,29 +230,73 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
-	renderSupportTab() {
+	renderContactUsCard( isPrimary = false ) {
 		return (
-			<div>
-				<Card className="theme__sheet-card-support">
-					<Gridicon icon="comment" size={ 48 } />
-					<div className="theme__sheet-card-support-details">
-						{ i18n.translate( 'Need extra help?' ) }
-						<small>{ i18n.translate( 'Visit the theme support forum' ) }</small>
-					</div>
-					<Button primary={ true } href={ getForumUrl( this.props ) }>Visit forum</Button>
-				</Card>
-				<Card className="theme__sheet-card-support">
-					<Gridicon icon="briefcase" size={ 48 } />
-					<div className="theme__sheet-card-support-details">
-						{ i18n.translate( 'Need CSS help? ' ) }
-						<small>{ i18n.translate( 'Visit the CSS customization forum' ) }</small>
-					</div>
-					<Button href="//en.forums.wordpress.com/forum/css-customization">Visit forum</Button>
-				</Card>
-			</div>
+			<Card className="theme__sheet-card-support">
+				<Gridicon icon="help-outline" size={ 48 } />
+				<div className="theme__sheet-card-support-details">
+					{ i18n.translate( 'Need extra help?' ) }
+					<small>{ i18n.translate( 'Get in touch with our support team' ) }</small>
+				</div>
+				<Button primary={ isPrimary } href={ '/help/contact/' }>Contact us</Button>
+			</Card>
+		);
+	},
+
+	renderThemeForumCard( isPrimary = false ) {
+		const description = isPremium( this.props ) ?
+			i18n.translate( 'Get in touch with the theme author' ) :
+			i18n.translate( 'Get help from volunteers and staff' );
+
+		return (
+			<Card className="theme__sheet-card-support">
+				<Gridicon icon="comment" size={ 48 } />
+				<div className="theme__sheet-card-support-details">
+					{ i18n.translate( 'Have a question about this theme?' ) }
+					<small>{ description }</small>
+				</div>
+				<Button primary={ isPrimary } href={ getForumUrl( this.props ) }>Visit forum</Button>
+			</Card>
 		);
 	},
 
+	renderCssSupportCard() {
+		return (
+			<Card className="theme__sheet-card-support">
+				<Gridicon icon="briefcase" size={ 48 } />
+				<div className="theme__sheet-card-support-details">
+					{ i18n.translate( 'Need CSS help? ' ) }
+					<small>{ i18n.translate( 'Get help from the experts in our CSS forum' ) }</small>
+				</div>
+				<Button href="//en.forums.wordpress.com/forum/css-customization">Visit forum</Button>
+			</Card>
+		);
+	},
+
+	renderSupportTab() {
+		const {
+			hasLoadedUserPurchasesFromServer,
```

Comment:
Do we need to use `hasLoadedUserPurchasesFromServer` or could we rely only on `isCurrentUserPaid`?